api_client/project_service_client.py

```diff
@@ -26,6 +27,17 @@ def run_analysis(self, analysis_id):
         job_info = response.json()
         result = job_info['jobId']
         return result
+    
+    def duplicate_analysis(self, analysis_id, payload):
+        url_path = f'/project/v1/analysis/{analysis_id}/duplicate'
+        url = urllib.parse.urljoin(self.service_base_url, url_path)
+
+        headers = self.session.get_auth_header()
+        headers["Content-Type"] = "application/json"
```

Comment:
Do we need to explicitly specify the Content and Accept header?

Reply:
Yes. Without specifying content-type and accept headers the post will fail with http code 415 Unsupported Media Type